webapp/components/Registration/CreateUserAccount.vue

```diff
@@ -74,12 +74,24 @@
             v-html="$t('termsAndConditions.termsAndConditionsConfirmed')"
           ></label>
         </ds-text>
+        <p>
+          <label>
+            <input id="checkbox1" type="checkbox" v-model="dataPrivacy" :checked="dataPrivacy" />
+            <span v-html="$t('components.registration.signup.form.data-privacy')"></span>
+          </label>
+        </p>
+        <p>
+          <label>
+            <input id="checkbox2" type="checkbox" v-model="minimumAge" :checked="minimumAge" />
+            <span v-html="$t('components.registration.signup.form.minimum-age')"></span>
+          </label>
+        </p>
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-15T21:38:05Z" title="Tuesday, October 15th 2019, 11:38:05 pm +02:00">Oct 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -74,12 +74,24 @@
             v-html="$t('termsAndConditions.termsAndConditionsConfirmed')"
           ></label>
         </ds-text>
+        <p>
+          <label>
+            <input id="checkbox1" type="checkbox" v-model="dataPrivacy" :checked="dataPrivacy" />
+            <span v-html="$t('components.registration.signup.form.data-privacy')"></span>
+          </label>
+        </p>
+        <p>
+          <label>
+            <input id="checkbox2" type="checkbox" v-model="minimumAge" :checked="minimumAge" />
+            <span v-html="$t('components.registration.signup.form.minimum-age')"></span>
+          </label>
+        </p>
```


We desparately need the checkbox form input.